config/redirects.yml

```
@@ -100,6 +100,7 @@ redirects:
   "/node/2339": "/how-to-apply-for-teacher-training/subject-knowledge-enhancement"
   "/node/2469": "/how-to-apply-for-teacher-training/subject-knowledge-enhancement"
   "/node/272": "/how-to-apply-for-teacher-training/subject-knowledge-enhancement"
+  "/how-to-apply-for-teacher-training/ske-explained-transcript": "/"
```

```suggestion
  "/how-to-apply-for-teacher-training/ske-explained-transcript": "/how-to-apply-for-teacher-training/subject-knowledge-enhancement"
```
would this be better redirecting to SKE page (not that important though as unlikely to be used!)